www/caddy/src/opnsense/service/conf/actions.d/actions_caddy.conf

```diff
@@ -35,3 +35,9 @@ command:/usr/local/sbin/pluginctl -s caddy status
 parameters:
 type:script_output
 message:Request Caddy status
+
+[showconfig]
```

Comment:
I'm all for simple one-word terms... `config` or `show` or `raw`.. config might be the most fitting but it's up to you to change or leave

Reply:
... and if there's more to show in the longrun `show.config` (split in two parameters) or `config.show`

Reply:
I think I'll go with ``config`` then, I don't know if there's more to show in the future.